modules/platforms/dotnet/Apache.Ignite.Core.Tests/JavaServer/pom.xml

```diff
@@ -35,7 +35,7 @@
         <dependency>
             <groupId>org.apache.ignite</groupId>
             <artifactId>ignite-core</artifactId>
-            <version>2.7.6</version>
+            <version>2.5.9</version>
```

Comment:
Central maven repo doesn't contain this professional release. It's ok since the version is replaced prior to a server startup, but could be confusing for new readers.

Reply:
Fixed, thanks